doc/kernel/usermode/mpu_stack_objects.rst

```diff
@@ -0,0 +1,49 @@
+.. _mpu_stack_objects:
+
+MPU Stack Objects
+#################
+
+Thread Stack Creation
+=====================
+
+Thread stacks are declared staticly with :c:macro:`K_THREAD_STACK_DEFINE()`
+or embedded within structures using c:macro:`K_THREAD_STACK_MEMBER()`
+
+For architectures which utilize memory protection unit (MPU) hardware,
+stacks are physically contiguous allocations.  This contiguous allocation
+has implications for the placement of stacks in memory, as well as the
```

Comment:
what are the implications?

Reply:
added a sentence pointing to the memory placement section later in the document.